src/vs/workbench/contrib/extensions/browser/extensionsList.ts

```diff
@@ -199,6 +202,18 @@ export class Renderer implements IPagedRenderer<IExtension, ITemplateData> {
 		data.name.textContent = extension.displayName;
 		data.author.textContent = extension.publisherDisplayName;
 		data.description.textContent = extension.description;
+
+		if (extension.local?.manifest.workspaceTrust?.required) {
```

Comment:
Seems like this will be shown everywhere where extension is showed. Eg: in extensions viewlet. Not sure if it is intentional, if so why do we want to show the description there where we do not have enough space ?

Reply:
It is appended everywhere but css hides it by default, and this is overridden in the workspace trust editor css to swap this description in for the extension description.